rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1822,17 +1822,21 @@ public Integer setLockWhitelistDisableBlockDelay(Transaction tx, BigInteger disa
         if (!isLockWhitelistChangeAuthorized(tx)) {
             return LOCK_WHITELIST_GENERIC_ERROR_CODE;
         }
-
         LockWhitelist lockWhitelist = provider.getLockWhitelist();
         if (lockWhitelist.isDisableBlockSet()) {
             return -1;
         }
-        int disableBlockDelay = disableBlockDelayBI.intValueExact();
-        int bestChainHeight = getBtcBlockchainBestChainHeight();
-        if (bestChainHeight < 0 || Integer.MAX_VALUE - disableBlockDelay < bestChainHeight) {
+        try {
+            int disableBlockDelay = disableBlockDelayBI.intValueExact();
+            int bestChainHeight = getBtcBlockchainBestChainHeight();
+            if (bestChainHeight < 0 || disableBlockDelay < 0 ) {
```

Comment:
Why would `bestChainHeight` be negative?

Reply:
my bad, i thougth that #697 was merged already